Swinject/Container+SwinjectStoryboard.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Container+SwinjectStoryboard.swift
+//  Swinject
+//
+//  Created by Yoichi Tagaya on 11/28/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+#if os(iOS) || os(OSX) || os(tvOS)
+extension Container {
+    /// Adds a registration of the specified view or window controller that is configured in a storyboard.
```

Comment:
Length Violation: Line should be 100 characters or less: currently 106 characters